keras/backend.py

```diff
@@ -3415,15 +3416,22 @@ def resize_images(x, height_factor, width_factor, data_format,
 
   if data_format == 'channels_first':
     x = permute_dimensions(x, [0, 2, 3, 1])
-  if interpolation == 'nearest':
-    x = tf.image.resize(
-        x, new_shape, method=tf.image.ResizeMethod.NEAREST_NEIGHBOR)
-  elif interpolation == 'bilinear':
-    x = tf.image.resize(x, new_shape,
-                                   method=tf.image.ResizeMethod.BILINEAR)
+  interpolations = {
+    'area': tf.image.ResizeMethod.AREA,
+    'bicubic': tf.image.ResizeMethod.BICUBIC,
+    'bilinear': tf.image.ResizeMethod.BILINEAR,
+    'gaussian': tf.image.ResizeMethod.GAUSSIAN,
+    'lanczos3': tf.image.ResizeMethod.LANCZOS3,
+    'lanczos5': tf.image.ResizeMethod.LANCZOS5,
+    'mitchellcubic': tf.image.ResizeMethod.MITCHELLCUBIC,
+    'nearest': tf.image.ResizeMethod.NEAREST_NEIGHBOR,
+  }
+  interploations_list = '`' + '`, `'.join(interpolations.keys()) + '`'
```

Comment:
Replace \` with "

Reply:
Changed in [Fix string formatting](https://github.com/keras-team/keras/pull/16249/commits/aa8af10a160845df30a8b28470e866e02c68bdf5)